squad/frontend/templates/squad/group.html

```diff
@@ -4,11 +4,10 @@
 {% load static %}
 
 {% block content %}
-<h1>{{group.name}}</h1>
-
-<p>
-{{group.description|default:""}}
-</p>
+<h2 class="page-header well">
+  {{group.name}}  <br>
+  <small>{{group.description|default:""}}</small>
```

Comment:
Description might be lengthy. Please limit the number of characters here or provide JS to expand/collapse the lengthy description.

Reply:
I saw sample code. I will fix it. What's the workflow here? Should I push a specific pull request just for this or add it as a patch to the future pull request?

Reply:
Adding a patch to this PR is OK. If the problem has to be fixed before merging (not in this case) you can amend the commit and force push to your branch (I'm using branches for PRs for this reason).